lib/compat/wordpress-6.0/class-wp-webfonts.php

```diff
@@ -165,6 +173,7 @@ public function validate_font( $font ) {
 			'font-feature-settings',
 			'font-variation-settings',
 			'line-gap-override',
+			'preload',
```

Comment:
Since `preload` is not a valid CSS prop, I think it would be best if we added it to the `// exceptions` section a few lines below, along with `provider`.
We'd also need to add a check in the local-provider class to make sure that no CSS is generated for this: https://github.com/WordPress/gutenberg/blob/9f5fb53ab959a9147f9a14dc1a610be38702c5b6/lib/compat/wordpress-6.0/class-wp-webfonts-provider-local.php#L193-L197